src/core/internal/createWebWorkerPromise.ts

```diff
@@ -8,11 +8,26 @@ interface createWebWorkerPromiseResult {
   worker: Worker
```

Comment:
This should be `itkWorker` instead of `Worker`

Reply:
We want to return the `Worker` interface. EWe only need it in the `itkWorker` form to do the assignment of the worker promise. The fact the `Worker` is augmented with the worker promise should be transparent to the user.

Reply:
I will fix up the lint issue